### PR TITLE
LPS-96235 Add a terms of interests provider on DXP

### PR DESCRIPTION
**Pull request by dgarciasarai**
Please, review naming :)

/cc @darquesdev

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/provider/AsahInterestTermsProviderTest.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+			RandomTestUtil.randomString(), RandomTestUtil.randomString()
+		};
+
+		Mockito.when(
+			_segmentsInterestTermsAsahCache.getInterestTerms(userId)
+		).thenReturn(
+			interestTerms
+		);
+
+		Context context = new Context();
```
Comment (darquesdev) [inline modules/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/provider/AsahInterestTermsProviderTest.java]: you don't need context, right?

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/cache/SegmentsEntryIdsAsahCache.java [hunk 2]
```diff
@@ -33,17 +33,18 @@
 @Component(
 	configurationPid = "com.liferay.segments.asah.connector.internal.configuration.SegmentsAsahConfiguration",
 	configurationPolicy = ConfigurationPolicy.OPTIONAL, immediate = true,
-	service = SegmentsAsahCache.class
+	service = SegmentsEntryIdsAsahCache.class
 )
-public class SegmentsAsahCache {
+public class SegmentsEntryIdsAsahCache {
```
Comment (darquesdev) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/cache/SegmentsEntryIdsAsahCache.java]: not sure about the name.. it depend on using AC context or DXP one

AC IndividualSegmentIds -> DXP SegmentsEntryIds

**Pull request closed without merging**